### PR TITLE
[T4A4][F12-A2] Lim Huan Hock

### PR DESCRIPTION
**Pull request by huanhock**

### CODE REVIEW

**Comment by huanhock** [comment]
ready for review

File: src/seedu/addressbook/data/person/Phone.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public static boolean isValidPhone(String test) {
     }
 
     @Override
-    public String toString() {
+    public String toSring() {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Phone.java:40]: I think this was accidental?

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A read-only immutable interface for a Person in the addressbook.
+ * Implementations should guarantee: details are present and not null, field values are validated.
+ */
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Printable.java:6]: Good job adding header comments.

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 3]
```diff
@@ -83,4 +84,17 @@ default String getAsTextHidePrivate() {
...
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+   default String getPrintableString(Printable... printables){
+       String printableString = "";
+       for (Printable printable : printables) {
+           printableString += printable.getPrintableString();
+           printableString += SEPARATOR;
+       }
+       printableString = printableString.substring(0, -2);
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java:97]: Perhaps you can look into `StringJoiner` in the java api to help in concatenating strings?

**Review by brandonyeoxg — COMMENTED** [review]
@huanhock Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**